File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonAuthors.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;
import java.util.List;

/**
 * Created by zawzaw on 22/03/18.
 */

public class GsonAuthors {

    @SerializedName("total")
    private int author_total;

    @SerializedName("data")
    private List<GsonAuthor> authorList;

    public int getAuthor_total() {
        return author_total;
    }

    public List<GsonAuthor> getAuthorList() {
        return authorList;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonAuthors.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 22/03/18.
 */

class GsonAuthors {

    @SerializedName("total")
    val author_total: Int = 0

    @SerializedName("data")
    val authorList: List<GsonAuthor>? = null

}
